packages/components/src/components/map-marker/map-marker.stories.ts

```diff
@@ -28,155 +36,95 @@ export const Default = {
-  },
-  render: () => {
-    return html`
-      <div class="flex gap-4 items-center">
-        <sd-map-marker variant="place">
-          <span class="slot slot--border slot--background">
-            <sd-icon name="content/image" color="primary"></sd-icon>
-          </span>
+      <div class="flex flex-col space-y-5">
+        <sd-map-marker state="default" variant="place">
+          <sd-icon name="content/image" color="primary"></sd-icon>
         </sd-map-marker>
-        <sd-map-marker variant="place">
-          <span class="slot slot--border slot--background">
-            <img class="" src="images/logo-unioninvestment-sm.svg" alt="Logo" />
-          </span>
+        <sd-map-marker state="hover" variant="place">
+          <sd-icon name="content/image" color="primary"></sd-icon>
         </sd-map-marker>
-        <sd-map-marker variant="cluster">
-          <span class="slot slot--border slot--background"> 8 </span>
+        <sd-map-marker state="active" variant="place">
+          <sd-icon name="content/image" color="primary"></sd-icon>
         </sd-map-marker>
       </div>
-    `;
-  }
+
+      <div class="flex flex-col space-y-5">
+        <sd-map-marker state="default" variant="cluster"> 8 </sd-map-marker>
+        <sd-map-marker state="hover" variant="cluster"> 8 </sd-map-marker>
+        <sd-map-marker state="active" variant="cluster"> 8 </sd-map-marker>
+      </div>
+    </div>
+  `
 };
 
 /**
- * Use the '--map-marker-scaling' css variable to scale the marker to your needs.
+ * Use the `animated` attribute to animate the marker.
  */
-export const Scale = {
-  parameters: {
-    controls: { exclude: ['default', 'state'] }
-  },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: {
-          type: 'template',
-          name: 'marker-scaling',
-          values: [
-            {
-              title: '--map-marker-scaling: 1.5',
-              value: '<div style="--map-marker-scaling: 1.5">%TEMPLATE%</div>'
-            },
-            {
-              title: '--map-marker-scaling: 2',
-              value: '<div style="--map-marker-scaling: 2">%TEMPLATE%</div>'
-            }
-          ]
-        }
-      },
-      constants: [
-        {
-          type: 'slot',
-          name: 'default',
-          value: '<sd-icon name="content/image" color="primary"></sd-icon>'
-        },
-        {
-          type: 'attribute',
-          name: 'variant',
-          value: 'place'
-        }
-      ],
-      args
-    });
-  }
+export const Animated = {
+  render: () => html`
+    <sd-map-marker class="animated-example" variant="main" state="default" animated=""></sd-map-marker>
+    <script>
+      const marker = document.querySelector('.animated-example');
+      setInterval(() => {
+        marker.animated = !marker.animated;
+      }, 2000);
+    </script>
+  `
+};
+
+/**
+ * Use the default slot to include content in the marker.
+ */
+export const Slot = {
```

Comment:
I would show it like this.
![image](https://github.com/user-attachments/assets/e9717004-0235-4bb1-8088-de7a804c0496)

Cluster should always contain the number of pins ... nothing else.

Reply:
Updated